src/harness/fake_sas.py

```diff
@@ -176,7 +176,16 @@ def _GetSuccessResponse(self):
   def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
-
+  def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
+    zone_data = request['zoneData']
+    if(zone_data['usage'] == 'EXCLUSION_ZONE'):
+      id_attributes = zone_data['id'].split('/')
+      zone_id = ''; 
+      for zone_id_part in id_attributes[4 : len(id_attributes)]:
+        zone_id += zone_id_part
+      return {'zone_id' : zone_id}
```

Comment:
Is the intent here is to re-create the zoneId from id field, by removing the '/'es starting from the 4th slash?